edgedb-errors/src/lib.rs

```diff
@@ -68,12 +68,90 @@
 //!    `.source()` chain but must not be swallowed, otherwise retrying
 //!    transaction may work incorrectly.
 //!
+//! # Nice Error Reporting
+//!
+//! We use [miette] crate for including snippets in your error reporting code.
+//!
+//! To make it work, first you need enable `fancy` feature in your top-level
+//! crate's `Cargo.toml`:
+//! ```toml
+//! [dependencies]
+//! miette = { version="5.3.0", features=["fancy"] }
+//! ```
```

Comment:
It seems the docs are missing `edgedb-errors = {version="...", features=["miette"]}`.

When I tried this branch, there were no fancy error messages at first. While investigating, I saw that the example sets the above feature.

I'd expect to be able to set this feature via `edgedb_tokio`. Otherwise, users have to add `edgedb-errors` as a direct dependency (which I anyway have done, but not everybody might do that).